cli/asc.js

```diff
@@ -466,8 +484,8 @@ exports.main = function main(argv, options, callback) {
     if (typeof features === "string") features = features.split(",");
     for (let i = 0, k = features.length; i < k; ++i) {
       let name = features[i].trim();
-      let flag = assemblyscript["FEATURE_" + name.replace(/-/g, "_").toUpperCase()];
-      if (!flag) return callback(Error("Feature '" + name + "' is unknown."));
+      let flag = assemblyscript[`FEATURE_${name.replace(/-/g, "_").toUpperCase()}`];
```

Comment:
Since this is a refactor the conversion to CONSTANT_SNAKE_CASE should become a function so it can also be used below and improves readability.